controller/login.go

```diff
@@ -46,14 +46,14 @@ func (c *LoginController) Login(ctx *app.LoginLoginContext) error {
 	return ctx.TemporaryRedirect()
 }
 
+// Callback implemnents the `GET /api/login/callback` endpoint
 func (c *LoginController) Callback(ctx *app.CallbackLoginContext) error {
 	state := ctx.Params.Get("state")
 	code := ctx.Params.Get("code")
 	redirectURL := rest.AbsoluteURL(ctx.RequestData, client.CallbackLoginPath(), nil)
-
 	redirectTo, err := c.app.AuthenticationProviderService().LoginCallback(ctx, state, code, redirectURL)
 	if err != nil {
-		ctx.ResponseData.Header().Set("Location", *redirectTo+"?error="+err.Error())
+		ctx.ResponseData.Header().Set("Location", redirectURL+"?error="+err.Error())
```

Comment:
@sbryzak can you confirm that we have to use the `redirectURL` value in case if error ? (the `*redirectTo` causes a panic since `redirectTo` is `nil` if an error occurs)

Reply:
I think that redirectURL makes more sense, yes.